packages/volto-slate/src/blocks/Text/SlashMenu.jsx

```diff
@@ -122,7 +125,7 @@ const PersistentSlashMenu = ({ editor }) => {
         hasAllowedBlocks
           ? allowedBlocks.includes(item.id)
           : typeof item.restricted === 'function'
-            ? !item.restricted({ properties, block: item })
+            ? !item.restricted({ properties, block: item, user: user })
```

Comment:
Reading the docs made me realize:
1. We need to pass navRoot and contentType here
2. We need to pass user where `restricted` is called in the BlockChooser: https://github.com/plone/volto/blob/main/packages/volto/src/components/manage/BlockChooser/BlockChooser.jsx#L60

Reply:
Done